RecoTauTag/HLTProducers/interface/L1TJetsMatching.h

```diff
@@ -111,26 +112,140 @@ std::pair<std::vector<T>, std::vector<T>> categorise(
 
   return output;
 }
+template <typename T>
+std::tuple<std::vector<T>, std::vector<T>, std::vector<T>> categoriseVBFPlus2CentralJets(
```

Comment:
Please move `categorise` and `categoriseVBFPlus2CentralJets` inside the class, as class member functions.
```suggestion
std::tuple<std::vector<T>, std::vector<T>, std::vector<T>> L1TJetsMatching::categoriseVBFPlus2CentralJets(
```

Reply:
done